packages/sakke/package.json

```diff
@@ -43,6 +44,7 @@
     "gulp-sourcemaps": "^3.0.0",
     "imagemin-pngcrush": "^6.0.0",
     "postcss-prefixwrap": "^1.25.0",
+    "prettier": "^2.5.1",
```

Comment:
🛑 Tää pitää olla devDepencency @saulirajala

Reply:
Fixed in https://github.com/valu-digital/npm-packages/commit/01664409b139abfc39e118e12f208ff12b35a377